src/manimShell.ts

```diff
@@ -657,6 +662,12 @@ export class ManimShell {
                         this.eventEmitter.emit(ManimShellEvent.IPYTHON_CELL_FINISHED);
                     }
 
+                    if (this.isExecutingCommand && data.match(IPYTHON_MULTILINE_START_REGEX)) {
+                        Logger.debug(`💨 IPython multiline detected, sending extra newline`);
+                        // \x7F deletes the extra line ("...:") from IPython
```

Comment:
Ok, that's funny, so it suffices to send a `DEL` (`\x7F`) to make IPython continue? But in contrary, appending a `\r\n` or `\n` or `\r` to the `checkpoint_paste()` command won't work? What a strange bug...

Reply:
Yes, it has to do something with the timing of sending the newline... If you paste something in IPython, it also does not execute it immediately (which is probably intended)